import timeit


def main():
    pass

if __name__ == "__main__":
    try:
        main()
    except Exception as err:
        print(f"[ERROR] {err}\n")
        exit(1)
